tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,6 @@
-#not sure what to do with sourceforge packages, instructions no longer in chocolatey wiki
 $packageName = 'swig'
-$url = 'http://downloads.sourceforge.net/project/swig/swigwin/swigwin-3.0.5/swigwin-3.0.5.zip'
+$url = "http://downloads.sourceforge.net/project/swig/swigwin/swigwin-$version/swigwin-$version.zip?r=&ts=1394566398&use_mirror=softlayer-ams"
 $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$version = '3.0.6'

```

Comment:
I know almost nothing about powershell...you can define variables *after* using them?